src/protocol/vm/tycho_simulation_contract.rs

```diff
@@ -193,10 +194,12 @@ where
 
 #[cfg(test)]
 mod tests {
-    use super::*;
-    use revm::primitives::{hex, AccountInfo, Address, Bytecode, B256, U256 as rU256};
     use std::str::FromStr;
 
+    use revm::primitives::{hex, AccountInfo, Address, Bytecode, B256, U256 as rU256};
+
+    use super::*;
+
```

Comment:
Hint: for these test sections if you import super first, then if you double import something that's already in super it will warn you. If you import super last it doesn't warn you.